### PR TITLE
docs: polish

### PR DESCRIPTION
**Pull request by jiacai2050**

### CODE REVIEW

File: docs/src/cn/design/storage.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ CeresDB 是一种基于 share-nothing 架构的分布式存储系统，不同服
 
 由于 memtable 不是实时持久化到底层存储系统，因此需要用 WAL 来保证 memtable 中数据的可靠性。
 
-另一方面，由于分布式架构的设计，要求 WAL 本身是高可用的，现在 CeresDB 中，主要有以下几种实现：
+另一方面，由于[分布式架构](architecture.md)的设计，要求 WAL 本身是高可用的，现在 CeresDB 中，主要有以下几种实现：
```
Comment (MachaelLee) [inline docs/src/cn/design/storage.md:25]: Is it better to link 'cluster.md'?

Comment (ShiKaiWi) [inline docs/src/cn/design/storage.md:25]:
```suggestion
另一方面，由于[分布式架构](clustering.md)的设计，要求 WAL 本身是高可用的，现在 CeresDB 中，主要有以下几种实现：
```

File: docs/src/en/design/storage.md [hunk 2]
```diff
@@ -24,11 +24,11 @@ A write request will be written to
 
 Since memtable is not persisted to the underlying storage system in real time, so WAL is required to ensure the reliability of the data in memtable.
 
-On the other hand, due to the design of the distributed architecture, WAL itself is required to be highly available. Now there are following implementations in CeresDB:
+On the other hand, due to the design of the [distributed architecture](architecture.md), WAL itself is required to be highly available. Now there are following implementations in CeresDB:
```
Comment (ShiKaiWi) [inline docs/src/en/design/storage.md:27]:
```suggestion
On the other hand, due to the design of the [distributed architecture](clustering.md), WAL itself is required to be highly available. Now there are following implementations in CeresDB:
```

File: docs/src/cn/design/storage.md (hunk 1 above)
Comment (jiacai2050) [reply docs/src/cn/design/storage.md]: done

**Review by MachaelLee — APPROVED** [review]
LGTM

**Pull request merged**